doc/Learn/Markup/toc.yml

```diff
@@ -32,6 +32,8 @@
       href: xref:Uno.Extensions.Markup.Storyboards
 - name: Using and Generating Extensions
   href: xref:Uno.Extensions.Markup.GeneratingExtensions
+- name: Using Theme Service
+  href: xref:Uno.Extensions.Markup.ThemeService
```

Comment:
This xref isn't matching the UID of the document

Reply:
@kazo0, @kazo0 this needs to be removed now if I am not mistaken right? 
Since we decided it should be placed in Learn/ThemeService, so no need to have this under Markup right?